config/cloud/blogs/luillyfe-Production-Grade-GKE-Storage-Provisioning-using-Cloud-SQL-0a5dfe3cefdf4ea5973998bf63e43949.json

```diff
@@ -0,0 +1,15 @@
+{
+  "expertise": "BEGINNER",
+  "version": 1,
+  "source": "other",
+  "authorIds": ["luillyfe"],
+  "author": "Fermin Blanco",
+  "title": "Notion – The all-in-one workspace for your notes, tasks, wikis, and databases.",
```

Comment:
Please consider, title: "Storage provisioning to a GKE Cluster"

Reply:
Thanks for the comment @luillyfe this is updated :)